network/core-node/admin-guide/commands.mdx

```diff
@@ -1,140 +1,76 @@
 ---
```

Comment:
Looks good! So admin guide just has the essential commands like new-db, run, catchup with a note to go to GH for rest.